### PR TITLE
'Train my mind to adapt to any circumstance.'

### PR DESCRIPTION
**Pull request by craigjbass**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+> “In this way you must understand how laughable it is to say, ‘Tell me what to do!’ 
+> What advice could I possibly give? 
+> No, a far better request is, 
+> ‘Train my mind to adapt to any circumstance.’ […] 
+> In this way, if circumstances take you off script […] 
+> you won’t be desperate for new prompting.“
+
+> – Epictetus, _Discourses_, 2.2.20b-1;24b-25a (~108 CE)
```
Comment (naliwajek) [inline README.md]: Hmm, this ain't adding new lines according to preview 🤔

Comment (craigjbass) [reply README.md]: image

Comment (naliwajek) [reply README.md]: Apparently we can add `</br>` tag at the end if we want to split them line by line:
```
> line 1 <br>
> line 2
```

Comment (craigjbass) [reply README.md]: I wanted to keep it fairly short on the site. I think this is ok

**Pull request merged**